src/makeRoutes.js

```diff
@@ -1,9 +1,13 @@
 import userRouter from './routes/user';
 import userAuthRouter from './routes/user/auth';
+import NotificationTemplateController from './controllers/NotificationTemplateController';
```

Comment:
Should import the router file for NotificationTemplates just like we are doing for Users, not the controller like you're doing.